### PR TITLE
fix race test for TestWatchChanges

### PR DESCRIPTION
**Pull request by xiaolanz**
address: https://github.com/istio/istio/pull/3780

go test --v ./mixer/pkg/runtime2/... --race -run TestWatchChanges

### CODE REVIEW

File: mixer/pkg/runtime2/listener_test.go [hunk 1]
```diff
@@ -18,10 +18,11 @@ import (
 	"errors"
 	"reflect"
 	"testing"
-	"time"
 
 	"github.com/gogo/protobuf/proto"
 
+	"sync"
```
Comment (GregHanson) [inline mixer/pkg/runtime2/listener_test.go]: file is not properly goimport'ed, `synch` should be between `reflect` and `testing`
standard is:

```
  go packages (alphabetized)
  <space>
  github packages (alphabetized)
  <space>
  istio packages (alphabetized)
```

Comment (xiaolanz) [reply mixer/pkg/runtime2/listener_test.go]: Done. PTAL

**Pull request merged**